protocol-units/da-sequencer/node/Cargo.toml

```diff
@@ -9,11 +9,16 @@ homepage.workspace = true
 publish.workspace = true
 rust-version.workspace = true
 
+[lib]
+name = "movement_da_sequencer_node"
+path = "src/lib.rs"
+
```

Comment:
Why would you need to specify this explicitly? This is the default name and top-level source file.